### PR TITLE
Rollback contract address

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
Flare network `OrderBook` and `RouteProcessorOrderBookV3ArbOrderTaker` need to match the ones provided under repo readme, as these are the contracts that are pinned and are indexed by the subgraph.

## Solution
Changed the contract addresses to match the pubstrat readme.

### CODE REVIEW

**Comment by thedavidmeister** [comment]
@hardyjosh @Siddharth2207 well actually i don't even think the readme should exist, shouldn't this actually be cross referenced against the settings file in our docs, and these addresses removed from the readme here?

**Comment by Siddharth2207** [comment]
> @hardyjosh @Siddharth2207 well actually i don't even think the readme should exist, shouldn't this actually be cross referenced against the settings file in our docs, and these addresses removed from the readme here?

Yes I agree, there should be only source of truth for loading settings.

**Review by thedavidmeister — CHANGES_REQUESTED** [review]
ok so it seems it wasn't clear that i was rejecting the PR earlier

please delete the readme so that it doesn't exist to contradict the strats and tests

**Comment by Siddharth2207** [comment]
> ok so it seems it wasn't clear that i was rejecting the PR earlier
> 
> please delete the readme so that it doesn't exist to contradict the strats and tests

Removed.

**Comment by Siddharth2207** [comment]
> ok so it seems it wasn't clear that i was rejecting the PR earlier
> 
> please delete the readme so that it doesn't exist to contradict the strats and tests

Removed.

File: src/tranche/tranche-space.rain [hunk 1]
```diff
@@ -38,7 +38,7 @@ subgraphs:
 
 orderbooks:
   flare-pubstrats:
-    address: 0x07701e3BcE4248EFDFc7D31392a43c8b82a7A260
+    address: 0xb06202aA3Fe7d85171fB7aA5f17011d17E63f382
```
Comment (thedavidmeister) [inline src/tranche/tranche-space.rain:41]: is this in all the pubstrats that need it?

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

**Pull request closed without merging**